packages/components/popper/src/popper.ts

```diff
@@ -1,23 +1,31 @@
 import { buildProps } from '@element-plus/utils'
 
-import type { ExtractPropTypes, PropType } from 'vue'
+import type { ExtractPropTypes } from 'vue'
 
 const effects = ['light', 'dark'] as const
 const triggers = ['click', 'contextmenu', 'hover', 'focus'] as const
 
 export const Effect = {
   LIGHT: 'light',
   DARK: 'dark',
-}
+} as const
+
+export const roleType = [
```

Comment:
How about replacing `roleType` with `roleTypes` ?

Reply:
LGTM, thanks for your advise. 

Yeah, we all do the same thing, we can merge first.